edpop-record-ontology.ttl

```diff
@@ -0,0 +1,140 @@
+# of also allowing other classes, like our own biographical records for authors.
+
+# The string literals are meant to be the best effort of EDPOP Explorer to give a 
+# somehow normalized, human-readable representation of the property, but they are not
+# thesaurized and certainly not guaranteed to be the same if they are conceptually the same
+# (e.g., Köln or Keulen for the city of Cologne).
+
+# My proposal would be to use these properties that can be extracted from the original record
+# in a deterministic way. That means: only string literals, and also links to other records
+# if they are defined in the original catalogues (such as links to CT Thesaurus).
+# Users will be able to enrich, correct and further normalize this data with annotations.
+# Annotations may also be used, perhaps, for automatic non-deterministic normalizations,
+# such as the place of publication as a link to Wikidata.
+
+edpoprec:title a rdf:property ;
+               skos:prefLabel "Title"@en ;
+               rdfs:domain edpoprec:BibliographicalRecord ;
+               rdfs:range xsd:string .
+
+edpoprec:alternativeTitle a rdf:property ;
+                            skos:prefLabel "Alternative title"@en ;
+                            rdfs:domain edpoprec:BibliographicalRecord ;
+                            rdfs:range xsd:string .
+
+# Or: contributor? And how to define the role of the contributor? Perhaps a separate
+# class is better. The catalogues generally describe the roles of the contributors quite
+# well (author, translator, illustrator, etc.)
+edpoprec:author a rdf:property ;
+                skos:prefLabel "Author"@en ;
+		rdfs:domain edpoprec:BibliographicalRecord ;
+		rdfs:range xsd:string .
+
+edpoprec:publisherOrPrinter a rdf:property ;
+                            skos:prefLabel "Publisher or printer"@en ;
+			    rdfs:domain edpoprec:BibliographicalRecord ;
+			    rdfs:range xsd:string .
+
+edpoprec:placeOfPublication a rdf:property ;
+                            skos:prefLabel "Place of publication"@en ;
+			    rdfs:domain edpoprec:BibliographicalRecord ;
+			    rdfs:range xsd:string .
+
+# Dating has xsd:string as its range, because the date is very often not a simple year
+# in the catalogues but can be anything, like [1650] for an estimation or a range like
+# 1800-1914.
+edpoprec:dating a rdf:property ;
+                skos:prefLabel "Dating"@en ;
+		rdfs:domain edpoprec:BibliographicalRecord ;
+		rdfs:range xsd:string .
+
+# For language, not sure which class to use. There are various IRI collections, of which
+# ISO 639-3 (e.g. https://iso639-3.sil.org/code/nld) and Glottolog
+# (e.g. https://glottolog.org/resource/languoid/id/mode1257) seem most attractive,
+# or perhaps Wikidata. Or just the ISO 639-3 code as a literal?
+edpoprec:language a rdf:property ;
+                  skos:prefLabel "Language"@en ;
+		  rdfs:domain edpoprec:BibliographicalRecord .
```

Comment:
Perhaps offer the ISO code as a property of the language, along with any literal description in the record. Then you can add more categorisations if you want, or choose which to represent in the interface.

ISO is pretty standardised and well-known so it is a good idea to include it. It is also the only one of these options that plays nicely with web accessibility.

However, ISO is probably not great for research.

At least for Dutch and English: since those languages underwent major shift in the early modern period, the ISO categorisation changes in that period. But since it describes a gradual change which is not consistently categorised in catalogues, whether a book is categorised as, say, `nl` or `dum`, is more a function of the catalogue than the content. The distinction mostly adds noise.

At the same time, ISO is quite bad at describing dialectal variation, which is much more relevant in older texts, though rarely described in metadata.

Reply:
The question how to represent languages is important and nontrivial, but the raw field should probably just be `xsd:string` like the others?

Reply:
Ah yes, this was some reflection the comment about different categorisations, but it makes sense that the raw value is a string.

Reply:
Aah, I had expected normalization of the language field to be quite trivial, based on the data that I had seen so far. But indeed it may turn out to be quite complicated. So let's use `xsd:string` for the raw `edpoprec:language` field, and think more about the normalized field.

@lukavdplas: what is the relation with web accessibility here?